packages/tooling/fast-tooling-react/src/form/custom-controls/control.css.utilities.syntax.tsx

```diff
@@ -0,0 +1,274 @@
+import React from "react";
+import { Syntax } from "@microsoft/fast-tooling/dist/css-data.syntax";
+import { RenderControlConfig } from "./control.css.utilities.props";
+import { renderDefault } from "./control.css.utilities";
+
+interface RenderSyntaxControlConfig extends RenderControlConfig {
+    syntax: Syntax;
+}
+
+/**
+ * The syntax control, for a list of syntaxes available refer to:
+ * https://github.com/mdn/data/blob/master/css/syntaxes.json
+ *
+ * These are provided from the @microsoft/fast-tooling package
+ * as TypeScript type.
+ */
+export function renderSyntaxControl(config: RenderSyntaxControlConfig): React.ReactNode {
+    switch (config.syntax) {
+        case "<absolute-size>":
```

Comment:
This switch case (as with the type switch case), though unused now, provides scaffolding for overrides as well as specifically identifying when custom web components will render (currently they all default to a textfield).

Reply:
Would it make sense to comment these out for now so at least they're tree-shaken?